opm/simulators/wells/VFPProdProperties.cpp

```diff
@@ -58,21 +58,32 @@ double VFPProdProperties::thp(int table_id,
                               const double& alq) const {
     const VFPProdTable* table = detail::getTable(m_tables, table_id);
 
-    //Find interpolation variables
-    double flo = detail::getFlo(aqua, liquid, vapour, table->getFloType());
-    double wfr = detail::getWFR(aqua, liquid, vapour, table->getWFRType());
-    double gfr = detail::getGFR(aqua, liquid, vapour, table->getGFRType());
+    // Find interpolation variables.
+    double flo = 0.0;
+    double wfr = 0.0;
+    double gfr = 0.0;
+    if (aqua == 0.0 && liquid == 0.0 && vapour == 0.0) {
+        // All zero, likely at initial state.
```

Comment:
I guess we still can get NAN if for instance only liquid is zero and we have VFPProdTable::WFR_WOR in getWFR etc. 
Should we guard against this as well or is this sufficient? 
An alternative is just to remove the error message and keep the code as is. It does not seem to harm to get 0 for wfr etc. when we have nans.

Reply:
I did not mean to prevent NAN in such cases, only in the obvious "we should not even try to evaluate the VFP table at zero rate" case. It is really mimimal. I think that we could run with this for a little while, then you know that if the error message shows up again you probably actually have a problem. Further, more invasive checking should be a new PR.

If you are happy with the current state, please merge.

Reply:
OK. I will merge.